l10n_ch_base_bank/tests/test_search_invoice.py

```diff
@@ -6,13 +6,23 @@
 
 
 @tagged("post_install", "-at_install")
-class TestSearchmove(common.SavepointCase):
+class TestSearchmove(common.TransactionCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
         cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
         cls.company = cls.env.ref("base.main_company")
-        cls.partner = cls.env.ref("base.res_partner_12")
```

Comment:
@sonhd91 why do we need this change? The given xml id has already been configured

Reply:
the `test_search_other_field` produce the search based on `partner_id`, and the old partner linked to many `account.move` (include the target one) so it make the test cases fail

Reply:
whatif adding `limit=1` to search's params?

Reply:
@sonhd91

Reply:
Yeah, `limit=1` help passed the test case indeed, but we need to add more param in search function, I think not quite stand up with the purpose of test case thought

Reply:
Dạ, okie anh